src/workerd/api/hibernatable-web-socket.c++

```diff
@@ -70,8 +70,7 @@ kj::Promise<WorkerInterface::CustomEvent::Result> HibernatableWebSocketCustomEve
     a.setHibernationManager(kj::addRef(KJ_REQUIRE_NONNULL(manager)));
   }
 
-  try {
-    co_await context.run(
+  auto promise = context.run(
```

Comment:
Can we keep this in the `try` block? I don't think it should be outside.

Reply:
The `try` is applied when we `co_await` the promise afterwards.